static-assets/components/cstudio-contextual-nav/status.js

```diff
@@ -175,12 +175,12 @@ CStudioAuthoring.ContextualNav.StatusNavMod = CStudioAuthoring.ContextualNav.Sta
 
             CStudioAuthoring.Service.getPublishStatus(CStudioAuthoringContext.site, {
               success: function (response) {
-
+                let message = (response.status.toLowerCase() === 'idle')? CMgs.format(contextNavLangBundle, 'ready') : CMgs.format(contextNavLangBundle, response.status.toLowerCase());
```

Comment:
A bit less repeated code this way:
```js
let message = CMgs.format(
  contextNavLangBundle, 
  (response.status.toLowerCase() === 'idle') ? 'ready' : response.status.toLowerCase()
);
```

Reply:
updated, also I added a new fn because the same message is called in other places.